src/widgets/merch-catalog/ui/merch-filter/merch-tags-toggle/merch-tags-toggle.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+import { MerchTagsToggle } from './merch-tags-toggle';
+
+describe('MerchTagsToggle', () => {
+  it('should render correctly in a close state', () => {
+    const handleClick = vi.fn();
+
+    render(<MerchTagsToggle isOpen={false} onClick={handleClick} />);
```

Comment:
```suggestion
describe('MerchTagsToggle', () => {
  const handleClick = vi.fn();
  
  it('should render correctly in a close state', () => {
    render(<MerchTagsToggle isOpen={false} onClick={handleClick} />);
```

and remove `const handleClick = vi.fn();` from all test-cases below